sql3/parser/ast.go

```diff
@@ -4187,10 +4229,14 @@ func (n *QualifiedTableName) String() string {
 		fmt.Fprintf(&buf, " %s", n.Alias.String())
 	}
 
-	if n.Index != nil {
-		fmt.Fprintf(&buf, " INDEXED BY %s", n.Index.String())
-	} else if n.NotIndexed.IsValid() {
-		buf.WriteString(" NOT INDEXED")
+	if n.With.IsValid() {
+		buf.WriteString(" WITH")
+		for i, o := range n.QueryOptions {
+			if i > 0 {
+				buf.WriteString(", ")
```

Comment:
I don't think you need the space after the comma, because the option will add a space before itself.

Reply:
Comma is OK, but I missed some parens...